python/lsst/ts/MTAirCompressor/aircompressor_csc.py

```diff
@@ -508,7 +539,7 @@ async def poll_loop(self):
                 return
 
 
-def _statusBits(fields, value):
+def _status_bits(fields, value):
```

Comment:
Consider renaming this function to something more clear. 

How about `convert_status_bit_to_bool`?

Also, there's no reason for you to mark this as "private" (the trailing "_") since this is not part of a class.

Furthermore, since this is not a CSC member, maybe move it to a `utils` module?

Reply:
Moved, I will add tests for this new function.